models/bob.js

```diff
@@ -3,11 +3,13 @@ const mongoose = require('mongoose');
 
 // Define and compile Bob Schema
 const bobSchema = mongoose.Schema({
-  data: {},
+  data:      {},
   startDate: { type: Date, default: Date.now() },
-  endDate: Date,
-  flavor: String,
-  tags: []
+  endDate:   Date,
+  flavor:    String,
+  tags:      [],
+  votes:     { type: Number, default: 1 },
```

Comment:
I like it start at 1 because then it defaults to +1 on the front page. +0 is a little sad.